File: 6kyu/DetectPangram.cs
// https://www.codewars.com/kata/545cedaa9943f7fe7b000048/

using System;
using System.Linq;

public static class Kata
{
  public static bool IsPangram(string str) {
    if (str.Length < 25) return false;
    
    string alphabet = "abcdefghijklmnopqrstuvwxyz";

    foreach (char c in alphabet) {
      if (str.ToLower().Contains(c)) continue;
      else return false;
    };
    return true;
  }
}

File: 6kyu/DetectPangram.js
// https://www.codewars.com/kata/545cedaa9943f7fe7b000048/ 

function isPangram(string){
    if (string.Length < 25) return false
    
    let alphabet = "abcdefghijklmnopqrstuvwxyz"

    for (let c of alphabet) {
      if (string.toLowerCase().includes(c)) {continue}
      else return false
    }
    return true
}
